UX4_ValidityReliability.ipynb

```diff
@@ -629,9 +635,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Task time and reaction time data often show positively skewed distributions, which means that arithmetic means are less suitable. As Sauro & Lewis argue, the geometric mean makes a better estimate (in particular for small sample sizes and the confidence interval calculations should be based on the t-value *for log transformed* data.\n",
+    "Task time and reaction time data often show positively skewed distributions, which means that arithmetic means are less suitable. As Sauro & Lewis argue, the geometric mean makes a better estimate (in particular for small sample sizes) and the confidence interval calculations should be based on the t-value for *log transformed* data.\n",
```

Comment:
Add the source to Sauro & Lewis. I suggest making the "Sauro & Lewis" clickable by doing [Sauro & Lewis](LINK)

Reply:
Good idea, can you incorporate that before I merge?

Reply:
It has now been added